android/expoview/src/main/java/host/exp/exponent/fcm/ExpoFirebaseMessagingDelegate.java

```diff
@@ -50,13 +50,13 @@ public void onMessageReceived(@NonNull RemoteMessage remoteMessage) {
         JSONObject manifest = new JSONObject(experienceDBObject.manifest);
         int sdkVersion = ABIVersion.toNumber(manifest.getString("sdkVersion")) / 10000;
 
-        // If an experience is on SDK newer than 39, that is SDK40 and beyond up till UNVERSIONED
-        // we only use the new notifications API as it is going to be removed from SDK40.
-        if (sdkVersion >= 40) {
+        // If an experience is on SDK 41 or above, use the new notifications API
+        // It is removed beginning with SDK41
+        if (sdkVersion > 40) {
```

Comment:
```suggestion
        if (sdkVersion >= 41) {
```

this matches more closely with the comment and imo is slightly easier to grok at a glance